vespajlib/src/main/java/com/yahoo/tensor/TensorType.java

```diff
@@ -59,6 +59,9 @@ public static Value largestOf(List<Value> values) {
 
         public static Value largestOf(Value value1, Value value2) {
             if (value1 == DOUBLE || value2 == DOUBLE) return DOUBLE;
+            if (value1 == FLOAT || value2 == FLOAT) return FLOAT;
```

Comment:
while this looks like it will promote int8->float, it won't work correctly in many cases, and now it's not longer doing "largestOf" anymore.  It's probably best to make this actually "largestOf" and do something else when type resolving the various operations.

Reply:
Thanks, fixed.